### DISCUSSION TITLE
Accuracy is incorrect when starting editor test mode from middle of a map (in non-osu gamemodes)

### DISCUSSION THREAD

**Discussion by hwabis** [ppy/osu#17256] [Q&A]
When I start test mode from the beginning of the map, the accuracy starts at 100% and is fine, but if I start in the middle of the map, the accuracy starts at 0% and climbs very slowly upward, as if I had missed all the notes before the timing point I started at.

Video of the issue:

https://user-images.githubusercontent.com/62194990/158302020-bba41769-251c-4f02-a71a-0195849e561e.mp4

Logs:
[runtime.log](https://github.com/ppy/osu/files/8249984/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8249985/updater.log)
[database.log](https://github.com/ppy/osu/files/8249986/database.log)
[performance.log](https://github.com/ppy/osu/files/8249987/performance.log)

This happens in taiko, catch the beat, and mania.

**Comment by smoogipoo** [comment]
Not sure what the expectation is here, and whether accuracy/score/combo should even be shown.

**Reply by hwabis** [reply]
To be clear, I just thought it was weird that osu! (standard) doesn't have this issue; the accuracy is correct, the hp bar starts at full instead of empty, and there's no lag spike when starting test mode.

**Reply by peppy** [reply]
I'd probably expect the gauges to start as if the play was perfect up until that point. Or at least start at same values (100% rather than 0% etc.)
